### ISSUE TITLE
[14.0] Account_invoice_import fails when using nline static product fails

### ISSUE THREAD

**Issue by bosd** [OCA/edi#572]
## Module
account_invoice_import

## Describe the bug
When importing an invoice with an configuration of nline static product. 
The import fails due to a function call which is renamed.

## To Reproduce
**Affected versions**: 14.0

1. Create a supplier
2. Create an import configuration for that supplier. Set to Nline Static Product
3. import an invoice file for that vender
4. See error message:

```

Odoo Server Error

Traceback (most recent call last):
  File "/opt/odoo/custom/src/odoo/odoo/addons/base/models/ir_http.py", line 237, in _dispatch
    result = request.dispatch()
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 685, in dispatch
    result = self._call_function(**self.params)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 361, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/opt/odoo/custom/src/odoo/odoo/service/model.py", line 94, in wrapper
    return f(dbname, *args, **kwargs)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 349, in checked_call
    result = self.endpoint(*a, **kw)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 914, in __call__
    return self.method(*args, **kw)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 533, in response_wrap
    response = f(*args, **kw)
  File "/opt/odoo/auto/addons/web/controllers/main.py", line 1398, in call_button
    action = self._call_kw(model, method, args, kwargs)
  File "/opt/odoo/auto/addons/web/controllers/main.py", line 1386, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/opt/odoo/custom/src/odoo/odoo/api.py", line 399, in call_kw
    result = _call_kw_multi(method, model, args, kwargs)
  File "/opt/odoo/custom/src/odoo/odoo/api.py", line 386, in _call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/opt/odoo/auto/addons/account_invoice_import/wizard/account_invoice_import.py", line 767, in import_invoice
    action = self.create_invoice_action(
  File "/opt/odoo/auto/addons/account_invoice_import/wizard/account_invoice_import.py", line 786, in create_invoice_action
    invoice = self.create_invoice(parsed_inv, import_config, origin)
  File "/opt/odoo/auto/addons/account_invoice_import/wizard/account_invoice_import.py", line 806, in create_invoice
    self.post_process_invoice(parsed_inv, invoice, import_config)
  File "/opt/odoo/auto/addons/account_invoice_import/wizard/account_invoice_import.py", line 992, in post_process_invoice
    il_vals = self._prepare_global_adjustment_line(
  File "/opt/odoo/auto/addons/account_invoice_import/wizard/account_invoice_import.py", line 890, in _prepare_global_adjustment_line
    account = amlo.get_invoice_line_account(
Exception

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 641, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 317, in _handle_exception
    raise exception.with_traceback(None) from new_cause
AttributeError: 'account.move.line' object has no attribute 'get_invoice_line_account'

```

**Expected behavior**
The invoice gets imported.

**Additional context**
Add any other context about the problem here. (e.g. OS, Python version, ...)
Likely caused by:
https://github.com/OCA/edi/blob/d2dfe88b6668530b62fa08e4fb73de73050e3019/account_invoice_import/wizard/account_invoice_import.py#L890

I think it should be renamed to:
`_get_computed_account`

**Comment by bosd** [comment]
Well I still hope it gets fixed

**Comment by alexis-via** [comment]
I confirm that the method get_invoice_line_account() was removed from the "account" module in odoo v13

**Comment by alexis-via** [comment]
_get_computed_account could be an option, but it doesn't exist in v16, so it's one of those methods that had a very short life !
So we'll use the underlying method get_product_accounts(), which has a longer life duration :)

**Comment by alexis-via** [comment]
Fixed. I tested it and it works now.

**Comment by bosd** [comment]
Fixed in #738
